pandas/io/excel/_base.py

```diff
@@ -878,12 +884,17 @@ class ExcelFile:
     engine : str, default None
         If io is not a buffer or path, this must be set to identify io.
         Supported engines: ``xlrd``, ``openpyxl``, ``odf``, ``pyxlsb``,
-        default ``xlrd``.
+        default ``xlrd`` for .xls* files, ``odf`` for .ods files.
         Engine compatibility :
-        - ``xlrd`` supports most old/new Excel file formats.
+        - ``xlrd`` supports most old/new Excel file formats but is no longer maintained.
```

Comment:
i think can you add a note that xlrd not support for python >= 3.9 (everywhere you mention)

Reply:
Sorry - I don't follow how this is different from python < 3.9, what do you mean by support? I am able to install xlrd on a python 3.9 environment:

![image](https://user-images.githubusercontent.com/45562402/100463250-86862580-3099-11eb-85ca-af4ef01000e4.png)

Reply:
see the issues when you actually use xlrd and pandas on 3.9. this is the entire reason for the deprecation; it is completely broken on 3.9

Reply:
I'm not seeing that:

```
Python 3.9.0 (default, Nov 15 2020, 14:28:56) 
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pandas as pd
>>> pd.__version__
'1.2.0.dev0+1364.g65319af6e'
>>> pd.DataFrame({'a': [1, 2, 3]}).to_excel('test.xlsx')
>>> pd.read_excel('test.xlsx', engine='xlrd')
   Unnamed: 0  a
0           0  1
1           1  2
2           2  3
```

From the issue, I see this in the OP

> xlrd is unmaintained and the previous maintainer has asked us to move towards openpyxl. xlrd works now, but might have some issues when Python 3.9 or later gets released and changes some elements of the XML parser, as default usage right now throws a PendingDeprecationWarning

but no other mention of Python 3.9 being broken.

Reply:
#37795